app/views/devise/confirmations/new.html.erb

```diff
@@ -22,7 +22,7 @@
       <button class="color-accent-brand text-underline cursor-pointer js-confirmation-button border-none p-0" role="button">Click here</button> if you didn't get the email...
     </div>
 
-    <%= inline_svg_tag("forem-background.svg", aria: true, title: "forem background", class: "absolute bottom-0 right-0 hidden m:block") %>
+    <%= inline_svg_tag("forem-background.svg", aria: true, title: "forem background", class: "forem-background absolute bottom-0 right-0 hidden m:block") %>
```

Comment:
We add a `forem-background` class so that we can set the `brand-accent` color for the background across all the views using it.